code/modules/admin/admin.dm

```diff
@@ -152,7 +152,7 @@ var/global/floorIsLava = 0
 				body += "<A href='?src=\ref[src];changehands=\ref[M]'>Change amount of hands (current: [M.held_items.len])</A> | "
 
 			// DNA2 - Admin Hax
-			if(iscarbon(M) && !isbrain(M) && !isalien(M))
+			if(iscarbon(M) && !isbrain(M) && !isalien(M) && !ismartian(M))
```

Comment:
Do slimes have DNA? A slime would pass this check.
If only humans and monkeys have DNA, it might be best to make this `if(ishuman(M) || ismonkey(M))`.